File: Android Source Code/PatientDetailMobileClient/src/com/mhise/model/Result.java
package com.mhise.model;

import java.io.Serializable;


/** 
*@(#)Result.java 
* @author R Systems
* @description This class contains All the getter and setter methods related to Result response
* @since 2012-08-10
* @version 1.0 
*/
public class Result implements Serializable{


	/**
	 * 
	 */
	private static final long serialVersionUID = -821818204042920523L;
	public	String ErrorCode;
	public  String ErrorMessage;		
	public String IsSuccess;
	
	public void setErrorCode(String ErrorCode)
	{
		this.ErrorCode = ErrorCode;
	}
	
	public void setErrorMessage(String ErrorMessage)
	{
		this.ErrorMessage = ErrorMessage;
	}
	
	public void setIsSuccess(String IsSuccess)
	{
		this.IsSuccess = IsSuccess;
	}
	
	
	public String getErrorCode()
	{
		return this.ErrorCode;
	}
	public String getErrorMessage()
	{
		return this.ErrorMessage;
	}
	public String getIsSuccess()
	{
		return this.IsSuccess;
	}
}

File: Mobius.Server/Mobius.CoreLibrary/Result.cs
using System;

namespace Mobius.CoreLibrary
{
    [Serializable]
    public class Result
    {
        private ErrorCode _errorCode;
        private string _errorMessage;
        private bool _IsSuccess = false;

        /// <summary>
        /// 
        /// </summary>
        public ErrorCode ErrorCode
        {
            get { return _errorCode; }
            set { _errorCode = value; }
        }

        /// <summary>
        /// 
        /// </summary>
        public string ErrorMessage
        {
            get { return _errorMessage; }
            set { _errorMessage = value; }
        }

        /// <summary>
        /// 
        /// </summary>
        public bool IsSuccess
        {
            get { return _IsSuccess; }
            set { _IsSuccess = value; }
        }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="errorCode"></param>
        public void SetError(ErrorCode errorCode)
        {
            this._errorCode = errorCode;
            this._errorMessage = Helper.GetErrorMessage(errorCode);
        }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="errorCode"></param>
        /// <param name="errorMessage"></param>
        public void SetError(ErrorCode errorCode, string errorMessage)
        {
            this._errorMessage = errorMessage;
            this._errorCode = errorCode;
        }
    }
}
